code/modules/reagents/chemistry/reagents/medicine_reagents.dm

```diff
@@ -891,9 +888,8 @@
 	overdose_threshold = 15
 	chemical_flags = REAGENT_CAN_BE_SYNTHESIZED
 	purity = REAGENT_STANDARD_PURITY
-	impure_chem = /datum/reagent/impurity/mannitol
+	inverse_chem = /datum/reagent/inverse
```

Comment:
Just `/datum/reagent/inverse`?

Reply:
Actually yeah, according to the wiki the inverse of mannitol is supposed to be toxic monomers. Should this be the _/datum/reagent/impurity/mannitol_ instead?

Reply:
Fuck it, I'm making the call that mannitoil as an inverse is more interesting.

Reply:
Huh ok